integration-cli/docker_cli_run_test.go

```diff
@@ -863,6 +863,10 @@ func TestRunEnvironmentErase(t *testing.T) {
 	// the container
 	cmd := exec.Command(dockerBinary, "run", "-e", "FOO", "-e", "HOSTNAME", "busybox", "env")
 	cmd.Env = []string{}
+	if os.Getenv("DOCKER_HOST") != "" { // retain DOCKER_TEST_HOST
+		cmd.Env = append(cmd.Env, fmt.Sprintf("DOCKER_HOST=%s", os.Getenv("DOCKER_HOST")))
```

Comment:
it will include an extra unix://var/run/docker.sock even though it's not needed by default. I'm not sure if that's the desired behavior.